research/conditional/train/configs/baselines/common.yaml

```diff
@@ -0,0 +1,27 @@
+time: "2-00:00:00"
+n_gpus: 1
+interactive_debug: false
+
+# train params
+params:
+  # base bass
+  mixed_precision: true
+
+  # logging etc.
+  use_neptune: true
+  project_name: "pmtest/llm-random"
+  name: "f-bench"
+  logging_interval_heavy: 5000
+  logging_interval_loss: 1000
+  save_weights_path: "weights" # <- do sprawdzenia
+  save_weights_interval: 1000 # <- do sprawdzenia (zapisywać na koniec też)
```

Comment:
I say `25_000`

Reply:
change save_weights_path to "weights.ckpt", it was my typo some time ago. Also it does not save weights at the end of the training (in practice it probably does, if number of iterations divides save_weights_interval).

Reply:
also, add neptune run id to yaml name

Reply:
No .ckpt suffixes allowed anymore. It handles step number already.